content/en/docs/tasks/administer-cluster/verify-signed-images.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Verify Signed Container Images
+content_type: task 
+min-kubernetes-server-version: v1.24
+---
+
+<!-- overview -->
+
+{{< feature-state state="alpha" for_k8s_version="v1.24" >}}
+
+## Pre-requisites
```

Comment:
Page looks very good!

Reply:
Overall this looks very good.  I have not seen the examples folder used before the way you used it. @sftim Is it ok how @PushkarJ stored his examples?

Reply:
This is being discussed in other comment threads. So resolving this and keeping others open for clarity on current state